frontstage/templates/surveys/surveys-share/recipient-email-address.html

```diff
@@ -77,17 +59,14 @@
             }}
         {% endcall %}
     {% endif %}
-
-    <h1>Enter recipient's email address</h1>
-    <p>We need the email address of the person who will be responding to your surveys.</p>
+    <div id="main_page_text">
+        <h1 class="ons-u-mb-s">New respondents email address</h1>
+        <p>We will send instructions to the email address that you provide.</p>
+        <p class="ons-u-mb-m">Once we confirm the new respondents access, they will be able to respond to the surveys you have selected.</p>
+    </div>
     {% set checkboxData = [] %}
     <form action="" method="post">
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-        {{
-            onsPanel({
-                "body": '<p>Make sure you have their permission to give us their email address.</p>'
-            })
-        }}
         {% if errors.email_address %}
             {% set errorEmailAddress = { "text": errors['email_address'][0],  "id": 'email_address_error' } %}
             {% set emailAddress = form.email_address.data %}
```

Comment:
The form header should read `New respondents email address` and not `Enter email address`